crates/scroll/payload/src/builder.rs

```diff
@@ -353,17 +359,26 @@ where
         &'a self,
         db: &'a mut State<DB>,
     ) -> Result<impl BlockBuilder<Primitives = Evm::Primitives> + 'a, PayloadBuilderError> {
+        // get the base fee for the attributes.
+        let base_fee: u64 = if self.chain_spec.is_curie_active_at_block(self.parent().number + 1) {
```

Comment:
This seems like an overkill since we will not run a pre-Curie chain. But we can keep it if you prefer.

Reply:
I can remove, as you prefer. I think it keeps the code more generic in case we need to run a dev net and simulate some pre-Curie chain, but as you want

Reply:
I see, we can keep it for now.